src/libs/actions/Task.ts

```diff
@@ -689,7 +689,14 @@ function setAssigneeValue(
             Onyx.merge(ONYXKEYS.PERSONAL_DETAILS_LIST, {[assigneeAccountID]: optimisticPersonalDetailsListAction});
         }
 
-        setAssigneeChatReport(report);
+        // The optimistic field may not exist in the existing report and it can be overridden by the optimistic field of previous report data when merging the assignee chat report
+        // So we should add these optimistic fields here to prevent merging incorrectly which can lead to creating duplicate created actions for an existing report
```

Comment:
```suggestion
        // Therefore, we should add these optimistic fields here to prevent incorrect merging, which could lead to the creation of duplicate actions for an existing report
```

Reply:
@nkdengineer a small suggestion for better understanding after we're good to go 😄

Reply:
@hungvu193 Updated.